.github/workflows/pkg.pr.new.yml

```
@@ -25,4 +32,6 @@ jobs:
         run: pnpm pkg set version=0.0.0-preview.${{ github.event.pull_request.head.sha }}
         working-directory: packages/nuqs
       - name: Publish to pkg.pr.new
-        run: pnpx pkg-pr-new publish --compact './packages/nuqs'
+      - if: steps.check-for-changes.outputs.skip != 'true'
```

Adding a `-` on the if causes it to become a new step, making the `- name` on the previous line a step without any action.

```suggestion
        if: steps.check-for-changes.outputs.skip != 'true'
```